WORKSPACE

```diff
@@ -238,3 +238,14 @@ http_file(
         "https://github.com/bazelbuild/bazel-toolchains/archive/44200e0c026d86c53470d107b3697a3e46469c43.tar.gz",
     ],
 )
+
+load("//container/ubuntu16_04/layers/bazel:version.bzl", "BAZEL_VERSION")
+
+http_file(
+    name = "bazel_installer",
+    sha256 = "7f6748b48a7ea6bdf00b0e1967909ce2181ebe6f377638aa454a7d09a0e3ea7b",
```

Comment:
Could we extract the sha and bazel version into a separate bzl file? Therefore it is easier for automatic trigger later. 

bazel/version.bzl would be a good place.

Reply:
done.